src/main/java/com/miraisolutions/xlconnect/Workbook.java

```diff
@@ -288,10 +292,17 @@ public boolean existsSheet(String name) {
         return workbook.getSheet(name) != null;
     }
 
-    public boolean existsName(String name) {
-        return workbook.getName(name) != null;
+    public boolean existsName(String name, String worksheetScope) {
```

Comment:
Similar to above, I guess there is a "global"/"any" scope which allows querying for a name on any worksheet?